WikiWatershed/model-my-watershed
# DRB Stream Layer

This adds a layer for the streams (as predicted by TauDem) in the DRB, and is only selectable when the DRB is visible.

![screen shot 2016-03-18 at 5 30 00 pm 2](https://cloud.githubusercontent.com/assets/1896461/13892641/22a0e4cc-ed2f-11e5-933a-d3a639a7f147.png)

##### Testing
* Copy this branch's version of `setupdb.sh` to a place accessible to psql (the app VM is good)
* Run `./setupdb.sh -d` to load data
* The DRB streams layer should only be selectable when the map viewport overlaps with the DRB.

Connects #1195
